style.css

```diff
@@ -1,455 +1,1107 @@
+  background: #fff;
+  border: 1px solid #cfd8dc;
 }
 
-.getmy:hover {
-  color: white;
+#get-in-touch {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px 16px;
+  width: 142px;
+  height: 48px;
+  background: #fff;
+  border-radius: 8px;
+  margin-top: 12px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+#get-in-touch:hover {
   background-color: #6070ff;
-  text-decoration: underline;
-  text-decoration-color: #396df2;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
+  color: #fff;
+}
+
+#get-in-touch:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+@media screen and (min-width: 768px) {
+  html,
+  body {
+    padding: 0;
+    margin: 0;
+    width: 100%;
+    overflow-x: hidden;
+  }
+
+  .sijijui img {
+    width: 8px;
+    height: 8px;
+  }
+
+  header {
+    display: flex;
+  }
+
+  .navbar .humburger {
+    display: none;
+  }
+
+  #nav-items {
+    display: flex;
+    gap: 15%;
+  }
+
+  .nav-item {
+    display: inline-block;
+    gap: 15%;
+  }
+
+  .nav-item a {
+    font-size: 1rem;
+    color: #000;
+    text-decoration: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 1.2rem;
+  }
+
+  .header-info-secion {
+    background-image: url(img/Header\ bg.png);
+    background-size: cover;
+  }
+
+  .main-section {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+  }
+
+  .greeting-hearder {
+    width: 100%;
+  }
+
+  .greeting-text {
+    width: 75%;
+    height: auto;
+  }
+
+  /* Desktop Work section */
+  .work {
+    display: grid;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 86.5%;
+    margin: 10px 0 0 28px;
+    height: auto;
+  }
+
+  .card-container {
+    display: flex;
+    flex-direction: row;
+    padding: 5%;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    margin-top: 100px;
+    height: 60vh;
+    width: 100%;
+  }
+
+  .project-img {
+    width: 32rem;
+    height: 28rem;
+    transition: transform 0.2s ease-out;
+    display: flex;
+    margin-right: 100px;
+  }
+
+  .jaribu {
+    display: flex;
+    margin-left: 2000px;
+  }
+
+  .project-img:hover {
+    transform: scale(1.1);
+  }
+
+  .wrap {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding-left: 2%;
+    margin-left: 10%;
+    width: 100%;
+    margin-top: 5rem;
+  }
+
+  .card-tittle {
+    margin-top: -15rem;
+    margin-left: -5.5rem;
+  }
+
+  .frame2-img {
+    margin-top: -1rem;
+    margin-left: -5.7rem;
+  }
+
+  .description {
+    margin-top: -0.25rem;
+    width: 85%;
+    margin-left: -5.3rem;
+  }
+
+  .languages-list {
+    margin-left: -5.3rem;
+    margin-top: -8.5rem;
+  }
+
+  .see-button {
+    margin-left: -5.4rem;
+  }
+
+  .order-2 {
+    order: 2;
+  }
+
+  /* About Section Desktop */
+  .about {
```

Comment:
- [ ] Kindly adhere to the project requirement as even mentioned by your previous code reviewer regarding your design not matching properly with the Figma design. Your design should start responsive from the `768px` screen size and I am aware that somewhere you are not able to understand the `Figma` design, I would suggest you to please don't copy the styles from `Figma`. Try to take just the design as the reference to build with the help of CSS. Right now because as I can see you are not using the Correct `layout`, `media-query` etc. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-10 19-59-41](https://github.com/Edgahkipkemoi/mobile/assets/66476329/b3f6e52d-3f74-411e-a069-bf00599b3a27)

</td>
<td>

![Screenshot from 2023-06-17 17-35-48](https://github.com/Edgahkipkemoi/mobile/assets/66476329/a78f13f4-6b2e-4ffd-8fa1-756f046495b5)

</td>
</tr>
</table>